src/subsystems/php/translators/composer-lock/default.nix

```diff
@@ -246,15 +245,15 @@ in {
       };
 
       # name of the default package
-      defaultPackage = composerJson.name;
+      defaultPackage = composerJson.name or "unknown";
```

Comment:
The `name` is already determined by the discoverer. I think we should just re-use `project.name` here.

Reply:
done